src/ray/object_manager/plasma/stats_collector.h

```diff
@@ -62,6 +75,7 @@ class ObjectStatsCollector {
 
   int64_t GetNumBytesCreatedCurrent() const;
 
+  CounterMap<ObjectStoreCounterType> bytes_by_loc_seal_;
```

Comment:
Consider `CounterMap<std::pair<LocationType, /*sealed*/bool>` or some variation--- imo, the 2x2 enum is a bit unexpected.